### DISCUSSION TITLE
Genymotion web down

### DISCUSSION THREAD

**Discussion by Langhere** [Genymobile/feedback#165] [Genymotion Desktop] [answered]
Any information about this? whay geny can't access and i can't download geny emulator

**Comment by jmaneyrol69** [comment, marked as answer]
@Langhere The website may have been under maintenance for upgrade. It is now fully accessible. Please try again.

**Reply by Langhere** [reply]
it still down sir, can't access

**Reply by jmaneyrol69** [reply]
@jmaneyrol69 I am sorry but our website works perfectly well. The problem is most probably on your side. Please contact our support at https://support.genymotion.com/hc/en-us/requests/new.
Thank you.
